### ISSUE TITLE
Crashes after latest Meteor update?

### ISSUE THREAD

**Issue by Herteby** [numtel/meteor-cfs-image-resize#5]
I just updated to from Meteor 1.3xx to 1.4xx and now image-resize crashes when attempting to upload an image.

```
Exception in callback of async function: TypeError: Cannot read property 'length' of null
     at Function.Buffer.concat (buffer.js:219:24)
     at packages/numtel_cfs-image-resize/server.js:35:1
     at runWithEnvironment (packages/meteor/dynamics_nodejs.js:110:1)
```

I know CFS is deprecated and not sure if you're still maintaining your package?

**Comment by StrongBearCeo** [comment]
I have the same problem

**Comment by StrongBearCeo** [comment]
I have created a PR for this issue, please check: https://github.com/numtel/meteor-cfs-image-resize/pull/6

**Comment by numtel** [comment]
Thanks @vhmh2005! I've released version 0.0.4 that includes the PR as well as updates JIMP to 0.2.27.

**Comment by StrongBearCeo** [comment]
Great!
